lib/deprecation_toolkit/rspec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
Where is this file supposed to be required from ? (Can't see any `require` call mentioning it)

Reply:
Sorry I'm a bit confused , why does a project using RSpec would need to require this file where as a project using minitest doesn't need to require anything for the toolkit  to work out of the box?
Also isn't it possible to "merge" the code in the `rspec_plugin.rb` file in here instead of having an indirection? (It seems that both files are meant to setup RSpec hooks?)

Reply:
I'm not 100% sure why this works this way but it seems like Rspec does not auto-load any config by default. We could require this file in the gem but Rspec wouldn't start using it unless we specifically tell it to. I did a bit of digging around but couldn't find any examples of an RSpec plugin where this wasn't the case. 

Pundit, as an example, uses the same method for integrating with RSpec: https://github.com/varvet/pundit#rspec

Same with FactoryBot. You need to require the right things in your test setup for it to work after you've installed them gem. https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md

We could potentially monkey-patch RSpec to include this by default but I'm not a big fan of that solution. If you're installing a gem that extends Rspec it's common practice to have to require something in your spec_helper file.

Reply:
Ok I see, thanks for the explanation. Can you add a small note in the Readme for Rspec user. You can squash your commit and I'll merge, thank you ❤️ !!

Reply:
Awesome! Thanks for all the patience and feedback along the way! Just finished rebasing but noticed an issue while I was doing some QA on this. Please hold off merging until I get it sorted!

Reply:
OK should be all good now! I testing this against Fullscript's rails app and have all the edge cases sorted out.

Readme is updated and should be good for another look when you get a chance.